pkg/image/source.go

```diff
@@ -43,6 +46,7 @@ var AllSources = []Source{
 	OciDirectorySource,
 	OciTarballSource,
 	OciRegistrySource,
+	PodmanDaemonSource,
```

Comment:
nit: I dunno if I believe in my next comment, so lots of grains of salt here... Since a popular podman installation configuration is socket-activation (and not a long running daemon) then should this be named more simply `PodmanSource`?

Reply:
from conversation: I can see this both ways, I'll resolve this for now.